agent/peering_endpoint_test.go

```diff
@@ -257,6 +292,76 @@ func TestHTTP_Peering_Read(t *testing.T) {
 	})
 }
 
+func TestHTTP_Peering_Delete(t *testing.T) {
+	if testing.Short() {
+		t.Skip("too slow for testing.Short")
+	}
+
+	t.Parallel()
+	a := NewTestAgent(t, "")
+
+	testrpc.WaitForTestAgent(t, a.RPC, "dc1")
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	// Insert peerings directly to state store.
+	// Note that the state store holds reference to the underlying
+	// variables; do not modify them after writing.
```

Comment:
This might be a copy-paste error, the write is going through the gRPC client and not directly into the state store 

I'm also not sure that the next two lines apply, given that it's not a direct state store write